src/main/java/cryptator/CryptaOperator.java

```diff
@@ -36,7 +36,6 @@ public enum CryptaOperator {
     GE(">=", (a, b) -> toBigInt(a.compareTo(b) >= 0), (a, b) -> a.ge(b)),
 
     AND(";", (a, b) -> toBigInt(!a.equals(BigInteger.ZERO) && !b.equals(BigInteger.ZERO)), (a, b) -> ((ReExpression) a).and((ReExpression) b));
```

Comment:
Don´t you think that display (console and graphviz)  would be more readable using `&&` rather than `;` for the operator ?

Reply:
I changed it.